### PR TITLE
Added support for messasge type

### PR DESCRIPTION
**Pull request by anilkbunkar**
Added support for messasge type whether it's `marketing` or `transactional`.

### CODE REVIEW

**Comment by anilkbunkar** [comment]
Hi @chetanmeh  @sawhneyh  @cmathis 
Can you please have a look?

**Review by mjasrotia — APPROVED** [review]

File: extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json [hunk 1]
```diff
@@ -51,6 +51,11 @@
           "title": "Journey Action ID",
           "type": "string",
           "description": "Journey Action ID, for which MessageExecution is triggered."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExecution/messageType": {
+          "title": "Message Type",
+          "type": "string",
+          "description": "Message Type, whether it's marketing or transactional message."
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json:58]: Should we define this field as enum?

**Comment by prabhum2** [comment]
@anilkbunkar -  Are you making enum changes as suggested by @sawhneyh or this PR is ready to me merged . Please Advise

File: extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json (hunk 1 above)
Comment (anilkbunkar) [reply extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json]: Thanks @sawhneyh 
I have made the changes. Please have a look.

**Review by sawhneyh — APPROVED** [review]

**Comment by anilkbunkar** [comment]
Hi @prabhum2 
We have a doubt, can we add more values in [hard-enum](https://github.com/adobe/xdm/blob/cf79e096f7f16270b133b0e8609d04b6faf7cb2f/CONTRIBUTING.md#soft-and-hard-enumerations) later if required? 
If yes, please merge the PR.

**Comment by anilkbunkar** [comment]
Hi @prabhum2 
Can you please merge the PR? 
I confirmed with @akshitjain that we can add more values to hard-enum if required later. 
Thanks.

**Pull request merged**